packages/mobile/src/utils/accountChecker.ts

```diff
@@ -0,0 +1,21 @@
+import { UnlockableWallet } from '@celo/wallet-base'
+import { call, select } from 'redux-saga/effects'
+import { AppEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { getWallet } from 'src/web3/contracts'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+export function* checkAccountExistenceSaga() {
```

Comment:
This is mostly a placeholder for now, right?

Reply:
It's meant to just send the event to analytics so that we can start tracking how many users are affected by this.